src/wasmMain/kotlin/OpenTest4Wasm.kt

```diff
@@ -0,0 +1,3 @@
+package com.willowtreeapps.opentest4k
+
+internal actual fun Throwable.className() = this::class.simpleName!!
```

Comment:
There are no qualified names in WASM (as of yet, possibly ever). This is similar to native where the qualified name is usually the same as the simple name.